lib/services/access_checker.go

```diff
@@ -321,24 +322,35 @@ func (a *accessChecker) CheckAccess(r AccessCheckable, state AccessState, matche
 func (a *accessChecker) GetKubeResources(cluster types.KubeCluster) (allowed, denied []types.KubernetesResource) {
 	if len(a.info.AllowedResourceIDs) > 0 {
```

Comment:
nit: I'd invert this `if` statement to reduce indentation for most of the function